rest/config.go

```diff
@@ -478,6 +478,9 @@ func readFromPath(path string, insecureSkipVerify bool) (rc io.ReadCloser, err e
 
 func (dbConfig *DbConfig) AutoImportEnabled() (bool, error) {
 	if dbConfig.AutoImport == nil {
+		if !dbConfig.UseXattrs() {
```

Comment:
Is this change required for this PR?  It seems unrelated.

Reply:
This change already exists on master and if I do a `git diff origin/master` I don't see this change so I don't know what github is doing here.